index.js

```diff
@@ -53,10 +59,15 @@ function format(text, opts) {
     ast.comments = [];
   }
   ast.tokens = [];
-  opts.originalText = text;
+  return ast;
+}
 
-  const printer = new Printer(opts);
-  return printer.print(ast);
+function getDocs(text, opts) {
+  return getPrinter(text, opts).getDocs(getAst(text, opts));
```

Comment:
I don't know if you want to do it in this PR, but I'd love to make `Printer` just a thing internally used in `printer.js` and just expose these `print` and `printToDocument` functions from `printer.js`. You don't have to do it here though.